ui/desktop/app/components/proxy-url/index.hbs

```diff
@@ -0,0 +1,34 @@
+<div class='proxy-url-container'>
+  <Hds::Dropdown @listPosition='bottom-left' as |dd|>
+    <dd.ToggleButton @color='secondary' @text={{this.selectedValue}} />
+    <dd.Interactive
+      @text={{t 'proxy-url.options.address-port'}}
+      {{on
+        'click'
+        (fn this.updateContent (t 'proxy-url.options.address-port') dd.close)
+      }}
+    />
+    <dd.Interactive
+      data-test-ssh-option
+      @text={{t 'proxy-url.options.ssh'}}
+      {{on
+        'click'
+        (fn this.updateContent (t 'proxy-url.options.ssh') dd.close)
+      }}
+    />
+  </Hds::Dropdown>
+  {{#if this.isAddressAndPort}}
+    <Hds::Copy::Snippet
```

Comment:
Are we expecting more types here?  

Can we do  `if this.isAddressAndPort` and `unless this.isAdressAndPort` for the ssh block?

Reply:
The intent is to have more options but I can update to this approach until that change comes.